test/fixture/assert.snap

```diff
@@ -0,0 +1,19 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
I think `jest-snapshot` just treats these files as CommonJS modules so rewriting the header should be harmless. Will do.